ext/standard/var.c

```diff
@@ -527,8 +527,13 @@ PHPAPI void php_var_export_ex(zval *struc, int level, smart_str *buf) /* {{{ */
 				buffer_append_spaces(buf, level - 1);
 			}
 
-			smart_str_append(buf, Z_OBJCE_P(struc)->name);
-			smart_str_appendl(buf, "::__set_state(array(\n", 21);
+			/* stdClass has no __set_state method, but can be casted to */
+			if (Z_OBJCE_P(struc) == zend_standard_class_def) {
+				smart_str_appendl(buf, "(object)array(\n", 15);
```

Comment:
[PHP documentation](https://secure.php.net/manual/en/language.types.type-juggling.php) uses space after cast, i.e. `(object) array(...)`.

Reply:
Alright, I decided to not be stubborn and accept the apparent overwhelming consensus here, and changed this when I rebased today. The manual is a particularly good argument.